### DISCUSSION TITLE
i have completed all knowledge checks still my module is not marked as complete

### DISCUSSION THREAD

**Discussion by Coder-shank** [github/india#556] [Students]
i have completed all knowledge checks still my module is not marked as complete,what's the issue it is still showing progess and not completed

**Comment by priyanshu688** [comment]
Yes, Same problem happening

**Reply by DevendraPhaniKumar** [reply]
I am facing same problem does anyone of your problem resolved.If so can you tell me how it resolved

**Comment by babita694** [comment]
me too facing same problem even how to get credit

**Reply by triad7** [reply]
Submit your certifications, by taking the Azure certifications, it will show completed.
